interfaces.go

```diff
@@ -0,0 +1 @@
+package gossamer
```

Comment:
Please could you explain why this needed.

Reply:
to avoid building errors like: 
`can't load package: package .: no Go files in /gocode/src/github.com/ChainSafe/gossamer`

Reply:
I don't see how this changes anything. Please explain or remove

Reply:
If I remove, the project wont compile.

Reply:
`go build` works fine when called from within `cmd/gossamer`. Running `go build` from the root is incorrect as there is no root package. `go build github.com/ChainSafe/gossamer/cmd/gossamer` should work from anywhere.

Reply:
yes, correct. I managed to remove it and build works fine now. Cheers.